backend/common/repo.py

```diff
@@ -306,31 +327,40 @@ def _get_release_index_from_file(self) -> typing.Dict[str, "DpkgRepo.ReleaseEntr
             elif os.access(self._get_parent_url(local_path, 0, "Release"), os.R_OK):
                 release_file = self._get_parent_url(local_path, 0, "Release")
             else:
+                logging.exception("No release file found in '{}'. Raising \
+                                   GeneralRepoException.".format(self._get_parent_url(local_path, 0)), exc_info=True)
                 raise GeneralRepoException("No release file found in {}".format(self._get_parent_url(local_path, 0)))
 
             try:
                 with open(release_file, "rb") as f:
                     release_file_content = f.read().decode("utf-8")
                     if self.gpg_verify and not self._has_valid_gpg_signature(local_path):
+                        logging.exception("GPG verfication failed: '{}'. \
```

Comment:
This one as well (`logging.error()`)

Reply:
Same as above. Also logs the stack trace.